src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -344,27 +364,11 @@ internal int TimeSpanToTimeOutMs(TimeSpan t)
         /// Create a copy of the connection info object.
         /// </summary>
         /// <returns>Copy of the connection info object.</returns>
-        internal virtual RunspaceConnectionInfo InternalCopy()
+        public virtual RunspaceConnectionInfo InternalCopy()
```

Comment:
Does it make sense to call this public API `InternalCopy()`?

Reply:
I think so.  It is used internally only (makes a copy used internally) and all connection info objects need to implement it.

Reply:
Since it's public, it's not so internal now, right?  I realize this will incur changes to internal usage, but it seems like the Internal prefix is no longer applicable

Reply:
`Clone()`?

Reply:
Ok.  Will update.